sonarts-sq-plugin/sonar-typescript-plugin/src/test/resources/mockEmptyTsLint.js

```diff
@@ -1 +1,8 @@
-throw "Some error message";
```

Comment:
@inverno can't we remove this test now?

Reply:
@vilchik-elena this test covers https://github.com/SonarSource/SonarTS/blob/4903183210ba64428413c67c012652f19f58cec3/sonarts-sq-plugin/sonar-typescript-plugin/src/main/java/org/sonar/plugin/typescript/ExternalTypescriptSensor.java#L206